cmd/exec/operations.go

```diff
@@ -497,22 +516,26 @@ func executeEvent(url string) (string, error) {
 		)
 
 		if err != nil {
-			return event["id"].(string), errors.New("filtetype not json")
+			return "", errors.New("direktiv server not reachable")
 		}
 
 		addAuthHeaders(req)
 
 		resp, err := http.DefaultClient.Do(req)
 		if err != nil {
-			return event["id"].(string), err
+			return "", err
 		}
 
 		if resp.StatusCode != http.StatusOK {
-			return event["id"].(string), errors.New("Bad request")
+			return "", errors.New("Bad request")
```

Comment:
Good practice in Go is for error messages to be all lowercase. That way they can be wrapped. For example, the caller of this function could print `log.Fatalf("Failed to send event: %v", err)`, resulting in `Failed to send event: bad request`.